.github/workflows/report_quiz/report_quiz.sh

```diff
@@ -46,14 +46,21 @@ main(){
         # We suppose that at this point we will have at least 2 options valid
         options=$([ -z "$A" ] || echo $(escp "$A"))$([ -z "$B" ] || echo ,$(escp "$B"))$([ -z "$C" ] || echo ,$(escp "$C"))$([ -z "$D" ] || echo ,$(escp "$D"))
         echo $options
+        
+        # We verify if the options don't exceed the max length
+        max_length=100
+        if [ ${#A} -gt $max_length ] || [ ${#B} -gt $max_length ] || [ ${#C} -gt $max_length ] || [ ${#D} -gt $max_length ]; then
+            echo "[skipped] one option exceed the max max_length $max_length"
```

Comment:
Is it possible to re-fetch for a new question/options when are in this case ?

Reply:
A new quiz should be submit instead of the quiz not skipped.

What do you think about create a function like propose_quiz who will be called in the main instead ?

```sh
main(){
    quiz_skipped=1
    # until the quiz is proposed correctly
    while [ $quiz_skipped -eq 1 ]; do
        propose_quiz
    done
}
```

Reply:
Sound good so far, what i propose you is : 
- having quiz_skipped as a global variable instead of having it inside the main
- and instead of equal to 1, i propose:
```bash
while [ $quiz_skipped -lt 3 ]; do
```
and inside `propose_quiz` i have an exit code 0 `exit 0` when everything went good, to tell the system, "stop right there, don't need for another iteration of the loop, i have successfully sent the quizz"

What do you think ?

Reply:
good

Reply:
I propose this #57

Reply:
Alright !
Since we're done with this PR and i approved it, can you squash and merge it @pythonbrad ?

Reply:
thanks